### PR TITLE
rpm: build ceph-resource-agents by default

### PR DESCRIPTION
**Pull request by smithfarm**
To align with debian build

Fixes: http://tracker.ceph.com/issues/17613
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
Jenkins re-test this please (unrelated rbd-mirror test failure)

File: ceph.spec.in [hunk 1]
```diff
@@ -14,7 +14,7 @@
 #
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 # 
-%bcond_with ocf
+%bcond_without ocf
```
Comment (tchaikov) [inline ceph.spec.in]: maybe the package name of "resource-agents" is too general, and there is an existing package named exactly the same, which is "Required" by our "resource-agents" as put by the "ceph.spec.in". see https://github.com/ClusterLabs/resource-agents/blob/master/resource-agents.spec.in and http://centos-packages.com/7/package/resource-agents/ . shall we rename it to ceph-resource-agent?

seems opensuse has renamed it already? https://rpmfind.net/linux/RPM/opensuse/updates/leap/42.2/oss/x86_64/ceph-resource-agents-10.2.4+git.1481215985.12b091b-4.1.x86_64.html @smithfarm is that you ? =)

Comment (smithfarm) [inline ceph.spec.in]: @tchaikov Actually, it already is called `ceph-resource-agents` due to the way RPM parses spec files. To make it be `resource-agents` there would need to be a `-n` in this line:

```
%package resource-agents
```

But, yes, in openSUSE the ocf build is enabled.

Comment (tchaikov) [inline ceph.spec.in]: very cool! thank you @smithfarm ! now i understand why we also package selinux now =D

Comment (tchaikov) [inline ceph.spec.in]: /me reading https://fedoraproject.org/wiki/How_to_create_an_RPM_package#Subpackages .

Comment (smithfarm) [inline ceph.spec.in]: @tchaikov Yes, and for example why the `ceph-osd` package is called that and not merely `osd` :-P

**Comment by smithfarm** [comment]
Pushed `wip-17613` to Shaman.

**Pull request merged**